### DISCUSSION TITLE
Module parse failed: Unexpected token (1:29).. help me..

### DISCUSSION THREAD

**Discussion by SEOKKAMONI** [vercel/turborepo#5336] [Help] [answered]
```
../../packages/global-style/src/font.ts
Module parse failed: Unexpected token (1:29)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> const fontGenerator = (weight: number, size: number, lineHeight: number, letterSpacing: number) => `
|     font-family: 'Pretendard Variable';
|     font-weight: ${weight};
```

I found an error in packages/global-style/src/font.ts while running after setting up the turbo repo Please help me..

this is my repo >> https://github.com/Bamdoliro/marururu/tree/feat/%2396

**Comment by anthonyshew** [comment, marked as answer]
The link to the repo is 404ing for me today. However, this error means that you have not set up your bundler correctly. Your TypeScript needs to be transpiled to JavaScript so it can be loaded into webpack.
